client/src/components/sections/whoAreWe/WhoAreWe.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
Good Job Nizar,
Great as usual, Just rename our files to index.jsx, style.css

Reply:
Thank you, Done!